src/skmultiflow/meta/oza_bagging.py

```diff
@@ -159,12 +159,13 @@ def partial_fit(self, X, y, classes=None, weight=None):
                 raise ValueError("The classes passed to the partial_fit function differ from those passed earlier.")
 
         self.__adjust_ensemble_size()
-
-        for i in range(self.n_estimators):
-            k = self.random_state.poisson()
-            if k > 0:
-                for b in range(k):
-                    self.ensemble[i].partial_fit(X, y, classes, weight)
+        r, _ = get_dimensions(X)
+        for j in range(r):
+            for i in range(self.n_estimators):
+                k = self.random_state.poisson()
+                if k > 0:
+                    for b in range(k):
+                        self.ensemble[i].partial_fit([X[j]], [y[j]], weight)
```

Comment:
There is a bug, `weight` is being passed as the third parameter which is `classes`

```suggestion
                        self.ensemble[i].partial_fit([X[j]], [y[j]], classes, weight)
```